tests/search/test_vespasearch.py

```diff
@@ -853,7 +856,12 @@ def test_csv_download_search_variable_limit(
     # monkeypatch.setattr(search, "_VESPA_CONNECTION", test_vespa)
     _populate_db_families(data_db)
 
-    query_spy = mocker.spy(search._VESPA_CONNECTION, "search")
```

Comment:
There are two instances of this mocker.spy in this file - do we need to do this for the other line too?

Reply:
Yep!